zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -300,13 +300,29 @@ def channel_message_to_zerver_message(constants: List[Any], channel: str,
 
     slack_usermention_regex = r"(<@)([a-zA-Z0-9]+)(\|)?([a-zA-Z0-9]+)?(>)"
 
-    # TODO mapping italics, bold, strikethrough etc
+    # Slack doesn't have mid-word message-formatting like Zulip.:w
```

Comment:
What does 'mid-word message-formatting' mean? E.g. ~wo~rd? It would be helpful to put in an example. Is there a reference for this?